chichiwang/exercise-js
# Add to Documentation page: Static Files, Error Boundary, Testing, Pull Request Template, Github Workflow

## Description
Linked to Issue: #65 
Add sections to the page `/documentation`:
* Static Files
* Error Boundary
* Testing
* Pull Request Template
* Github Workflow

Additionally, make grammatical fixes and update code snippets for sections in `docs/README.md`:
* Static Files
* Error Boundary
* Testing

## Changes
* Add new colors to `$code-colors` map in `app/var.module.scss`:
  * `darkBlue`
  * `grey`
  * `purple`
* Grammatical fixes to sections in `docs/README.md`
  * Static Files
  * Error Boundary
  * Testing
* Code snippet update in `docs/README.md` in section "Error Boundary"
  * Change the order of import statements
* Create component `pages/Documentation/ErrorBoundary.tsx`
* Create component `pages/Documentation/GithubWorkflow.tsx`
* Create component `pages/Documentation/PullRequestTemplate.tsx`
* Create component `pages/Documentation/StaticFiles.tsx`
* Create component `pages/Documentation/Testing.tsx`
* Add sections to `pages/Documentation/index.tsx`
  * Static Files
  * Error Boundary
  * Testing
  * Pull Request Template
  * Github Workflow

## Steps to QA
* Verify the changes to `docs/README.md` look correct
* Pull down and switch to this branch
* Verify in browser on the `/documentation` page:
  * New content looks correct
  * Links are correct
  * Copying from the code block in section "Error Boundary" copies the code to system clipboard
  * Print styles (print preview) looks correct
